src/test/java/cz/sparko/boxitory/test/e2e/versionastimestamp/VersionAsTimestampTest.java

```diff
@@ -0,0 +1,45 @@
+package cz.sparko.boxitory.test.e2e.versionastimestamp;
+
+import cz.sparko.boxitory.test.e2e.AbstractIntegrationTest;
+import org.springframework.test.context.TestPropertySource;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.Timestamp;
+
+import static org.hamcrest.Matchers.is;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@TestPropertySource(properties = {"box.version_as_timestamp=true"})
+public class VersionAsTimestampTest extends AbstractIntegrationTest {
```

Comment:
create pls second e2e test that checks when there is already some description for the box, that date is prepended

Reply:
you can simply create two boxes here and ```descriptions.csv``` file with description for just one box.

Reply:
Done